### ISSUE TITLE
Model required scaling down when using bigger nozzle

### ISSUE THREAD

**Issue by Chronoless** [daid/LegacyCura#972]
Hi,

I'm trying to print a vase-like object which is supposed to just fit the UM1 build size.
I use bigger nozzles with my UM1 and I noticed that I have to scale down the model to have Cura start slicing when I use bigger nozzles.
When in Cura I use .4 Nozzle, .4 Shell, No Fill, .2 Layer Height I need to scale down to .95 which makes the object 186.3 round, 151.0 high.
When I use the following settings: .8 Nozzle, .8 Shell, No Fill, .4 Layer I need to scale down to .87 which makes the object 170.7 round 139.0 high.
When I then change the Shell Thickness to 1.0 I need to scale down to .83 which makes the object 162.8 round and 132.6 high.

I'm using Cura 14.09 Mac.

Why do I have to scale down the object to be able to print it while the only difference is the size of the Nozzle, the Layer Height and Shell Thickness?

**Comment by daid** [comment]
Got brim/skirt enabled?

**Comment by Chronoless** [comment]
Jup, that was it! The brim was far away from the edges though. But I guess the printability of the model is calculated using the extreme dimensions + brims of the model and not the actual brim?

**Comment by daid** [comment]
Yes, that's because the frontend does not know the actual bottom layer dimension+brim.
